extension/storage/redisstorage/config.go

```diff
@@ -0,0 +1,18 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package redisstorage // import "github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage/redisstorage"
```

Comment:
The existing storage extensions (dbstorage, filestorage) are one of the few extensions whose package name does not end with `extension` (https://github.com/open-telemetry/opentelemetry-collector-releases/blob/v0.104.0/distributions/otelcol-contrib/manifest.yaml#L36-L37). Should we continue this pattern? Isn't this a mistake that should be fixed in those extensions and not replicated in a new component?

```suggestion
package redisstorageextension // import "github.com/open-telemetry/opentelemetry-collector-contrib/extension/storage/redisstorageextension"
```